pkg/controller/migmigration/backup.go

```diff
@@ -124,6 +124,25 @@ func (t *Task) getStageBackup() (*velero.Backup, error) {
 }
 
 func (t *Task) getPodVolumeBackupsForBackup(backup *velero.Backup) *velero.PodVolumeBackupList {
+	nl := map[string]string{
+		"velero.io/backup-name": backup.Name,
+	}
+
+	client, err := t.getSourceClient()
+	if err != nil {
+		log.Trace(err)
+	}
+	list := velero.PodVolumeBackupList{}
+	err = client.List(
+		context.TODO(),
+		k8sclient.MatchingLabels(nl),
+		&list)
+	if err != nil {
+		log.Trace(err)
+	}
+	if len(list.Items) > 0 {
```

Comment:
@JaydipGabani no need to check for length of list. We would just return the &list. It will be nil